apps/web-giddh/src/app/financial-reports/components/profit-loss/profit-loss.component.ts

```
@@ -59,120 +65,142 @@ export class ProfitLossComponent implements OnInit, AfterViewInit, OnDestroy {
-                        g.isCreated = true;
-                        g.isIncludedInSearch = true;
-                        g.isOpen = true;
-                        g.childGroups.forEach(c => {
-                            c.category = "income";
-                            c.isVisible = true;
-                            c.isCreated = true;
-                            c.isIncludedInSearch = true;
-                        });
-                    });
-                }
+        if (data && data.expArr) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)